TCMSApp/src/client/app/tests/tests.controller.js

```diff
@@ -10,7 +10,13 @@
     function TestsController(logger, FakeTestsFactory, $uibModal) {
         var vm = this;
 
+        vm.stepList = [1,2];
+        vm.addStep = addStep;
+        vm.delStep = delStep;
+
         vm.trigger = 0; // suite -> testCases
+        vm.checked = false; // checkbox for testCases
+        vm.open = open; // open modal for new suite
```

Comment:
openAddSuite or something like that is better (imho) 
Pros:
- self-describing
- you shall not refactor code when you will add some more modals to this view